ale_linters/elixir/credo.vim

```diff
@@ -32,6 +32,6 @@ endfunction
 call ale#linter#Define('elixir', {
 \   'name': 'credo',
 \   'executable': 'mix',
-\   'command': 'mix credo suggest --format=flycheck --read-from-stdin %s',
+\   'command': 'MIX_ENV=lint mix credo suggest --format=flycheck --read-from-stdin %s',
```

Comment:
For windows, this will need to be `set MIX_ENV=lint && ...` Look at say `ale_linters/slim/slimlint.vim` for an example of setting environment variables for all supported platforms.

Reply:
Thanks @w0rp . But I'll need more time to incorporate that and test it on windows. Feel free to close this PR if you'd like to. I can open another one when it's ready.